src/test/java/org/commcare/formplayer/tests/MediaCaptureTest.kt

```diff
@@ -235,6 +237,43 @@ class MediaCaptureTest {
         }
     }
 
+    @Test
+    fun testPurgeMedia() {
+        val formResponse = startImageCaptureForm()
+        val responseBean: FormEntryResponseBean
+        try {
+            responseBean = saveImage(formResponse, "media/valid_image.jpg", "valid_image.jpg")
+        } catch (e: Exception) {
+            fail("Unable to save a valid file due to " + e.message)
+        }
+
+        // get file from path and check if it's the same file
+        var expectedFilePath = getExpectedMediaPath(formResponse.session_id, responseBean)
+        var originalSavedFile = expectedFilePath.toFile()
+        assertTrue("Could not find saved file on the filesystem", originalSavedFile.exists())
+
+        // check that metadata was created and values match expected
+        val fileName = expectedFilePath.fileName.toString()
+        val metadataId = fileName.substring(0, fileName.indexOf("."))
+        val metadata = mediaMetaDataService.findById(metadataId)
+        assertEquals(metadata.formSession.id, formResponse.session_id)
+
+        val fis = FileUtils.getFileStream(this.javaClass, expectedFilePath.toString())
+        val file = MockMultipartFile(PART_FILE, fileName, MediaType.IMAGE_JPEG_VALUE, fis)
+
+        metadata.formSession = null
+        mediaMetaDataService.saveMediaMetaData(metadata)
```

Comment:
we should simplify this by omitting the checks that we are already doing as part of `testImageCapture_fileSaveAndReplace`. We onlly want to test purge as part of this test, so we should only retain the code that's necessary for `purge` testing below and remove the additional checks.